### PR TITLE
[dashboard] Add more navigation possibilities during stopping and stopped

### PR DESCRIPTION
**Pull request by svenefftinge**
- During stopping I should be able to go back to the dashboard.
- The context URL in the workspace description element is now a link.

### CODE REVIEW

File: components/dashboard/src/start/StartWorkspace.tsx [hunk 1]
```diff
@@ -206,7 +206,7 @@ export default class StartWorkspace extends React.Component<StartWorkspaceProps,
       // clone or backup download). After this phase one can expect the workspace to either be Running or Failed.
       case "initializing":
         phase = StartPhase.Starting;
-        statusMessage = <p className="text-base text-gray-400">{isPrebuilt ? 'Loading prebuild …' : 'Cloning repository …'}</p>;
+        statusMessage = <p className="text-base text-gray-400">{isPrebuilt ? 'Loading prebuild …' : 'Initializing content …'}</p>;
```
Comment (jankeromnes) [inline components/dashboard/src/start/StartWorkspace.tsx:209]: Why use a less transparent expression here? As a user, if I see "Initializing content..." I don't know what's happening and assume Gitpod is just slow. If I see "Cloning repository..." I may think "ah, right, my repository is pretty large, that's why it's taking a while".

**Review by jankeromnes — APPROVED** [review]
Many thanks! Looks good to me, except for the less transparent loading message.

File: components/dashboard/src/start/StartWorkspace.tsx (hunk 1 above)
Comment (svenefftinge) [reply components/dashboard/src/start/StartWorkspace.tsx:209]: Seeing `cloning repository` on restarting workspaces is wrong.

Comment (gtsiolis) [reply components/dashboard/src/start/StartWorkspace.tsx:209]: Cloning repository also sounds friendlier but let's go with _Initiazition_ if cloning inaccurate when restarting workspaces. We can always refine this later if needed. Thanks @jankeromnes @svenefftinge! 💭

**Pull request merged**